lib/elixir/lib/io.ex

```diff
@@ -207,7 +207,7 @@ defmodule IO do
     :elixir_errors.warn([to_chardata(message), ?\n])
   end
   def warn(message, stacktrace) when is_list(stacktrace) do
-    :elixir_errors.warn([to_chardata(message), ?\n, Exception.format_stacktrace(stacktrace)])
+    :elixir_errors.warn([to_chardata(message), ?\n, Exception.format_stacktrace(stacktrace, "  ")])
```

Comment:
Let's just call map_join here with format_stacktrace_entry. :) This way we don't need to add indent to format_stacktrace :)